server/channels/app/platform/web_hub.go

```diff
@@ -497,6 +502,7 @@ func (h *Hub) Start() {
 				for _, webConn := range connIndex.ForUser(userID) {
 					webConn.InvalidateCache()
 				}
+				connIndex.InvalidateCMCacheForUser(userID)
```

Comment:
To confirm, this is the only place where we need to invalidate, and it would trigger if a user joined a new channel or left an existing one (and a few more cases).

Reply:
Correct. We are just doing it at the same level as the `webConn.InvalidateCache`, because we are basically hoisting up the membership check from webConn level to webHub level.